### DISCUSSION TITLE
Is there a possibility to tell Velero not to backup volumes?

### DISCUSSION THREAD

**Discussion by Aracki** [vmware-tanzu/velero#4784] [Community support Q&A] [answered]
It's not clear from [the documentation](https://velero.io/docs/v1.8/customize-installation/) if there is a flag for Velero not to make Volume backups/snapshots. 

My use-case is to backup only manifests of all K8s objects.

**Comment by doverk96** [comment]
@Aracki Searching for the same. Please do update here if you are able to go through it.

**Comment by Aracki** [comment, marked as answer]
I realized you can create Velero Schedules with the --snapshot-volumes=false flag. This way you prevent Velero to create snapshots.
